src/lib/commands/config.ts

```diff
@@ -72,16 +84,18 @@ export const registerCommands = (context: ExtensionContext) => {
     commands.registerCommand(
       COMMANDS.CONFIG.SELECT_DEFAULT_ACCOUNT,
       async () => {
-        const config = getConfig();
-        if (config && config.portals) {
+        const defaultAccount: string | number | null | undefined =
+          getConfigDefaultAccount();
```

Comment:
`getConfigDefaultAccount` has its own return type, so it's redundant to declare the types for `defaultAccount` here.